<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>resultsForm</class>
 <widget class="QWidget" name="resultsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>772</width>
    <height>422</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>772</width>
    <height>422</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>772</width>
    <height>422</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Puzzle stacker</string>
  </property>
  <widget class="QPushButton" name="back_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Powrót</string>
   </property>
  </widget>
  <widget class="QPushButton" name="next_button">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>360</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Następny</string>
   </property>
  </widget>
  <widget class="QLabel" name="show_image">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>710</width>
     <height>340</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>710</width>
     <height>340</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>710</width>
     <height>340</height>
    </size>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;TextLabel&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="previous_button">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>360</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Poprzedni</string>
   </property>
  </widget>
  <widget class="QPushButton" name="main_button">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>360</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Powrót do menu</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="prompt_image.qrc"/>
 </resources>
 <connections/>
</ui>
